### PR TITLE
feature/20612 SQL Session store for Express

### PR DESCRIPTION
**Pull request by GuyHarwood**

### CODE REVIEW

File: admin/data/sql/migrations/077.do.sql-session-store.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE [mtc_admin].[sessions](
+  [sid] varchar(255) NOT NULL
```
Comment (jon-shipley) [inline admin/data/sql/migrations/077.do.sql-session-store.sql]: If this is 255 chars and un-indexed won't retrievals be very very slow?

Comment (GuyHarwood) [reply admin/data/sql/migrations/077.do.sql-session-store.sql]: its the primary key

Comment (GuyHarwood) [reply admin/data/sql/migrations/077.do.sql-session-store.sql]: its the primary key

File: admin/data/sql/migrations/077.do.sql-session-store.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    CONSTRAINT [pk_sessions] PRIMARY KEY CLUSTERED ([sid] ASC),
+  [expires] datetimeoffset NOT NULL,
```
Comment (jon-shipley) [inline admin/data/sql/migrations/077.do.sql-session-store.sql]: Do we want to index the `expires` field - then we can run a query based on this col.

File: docker-compose.admin-test.yml [hunk 3]
```diff
@@ -7,14 +7,11 @@ services:
     - "3001:3001"
     environment:
       SQL_SERVER: "sqldb"
-      MONGO_CONNECTION_STRING: "mongodb://mongo/mtc"
```
Comment (jon-shipley) [inline docker-compose.admin-test.yml]: 😀

File: admin/data/sql/migrations/077.do.sql-session-store.sql (hunk 2 above)
Comment (GuyHarwood) [reply admin/data/sql/migrations/077.do.sql-session-store.sql]: given the overhead of indexing millisecond precision dates i would rather not.  The situations where we would need to query this are minimal, and would likely be done offline.

if it were a common lookup we could look at that.

**Pull request merged**